library/src/test/java/com/chuckerteam/chucker/internal/support/FormatUtilsTest.kt

```diff
@@ -58,4 +64,80 @@ class FormatUtilsTest {
             """.trimIndent()
         )
     }
+
+    @Test
+    fun testFormatHeaders_withNullValues() {
+        val result = FormatUtils.formatHeaders(null, false)
+        assertThat(result).isEmpty()
+    }
+
+    @Test
+    fun testFormatHeaders_withEmptyValues() {
+        val result = FormatUtils.formatHeaders(listOf(), false)
+        assertThat(result).isEmpty()
+    }
+
+    @Test
+    fun testFormatHeaders_withoutMarkup() {
+        val result = FormatUtils.formatHeaders(exampleHeadersList, false)
+        val expected = "Accept: text/html\nAuthorization: exampleToken\n"
+        assertThat(result).isEqualTo(expected)
+    }
+
+    @Test
+    fun testFormatHeaders_withMarkup() {
+        val result = FormatUtils.formatHeaders(exampleHeadersList, true)
+        val expected = "<b> Accept: </b>text/html <br /><b> Authorization: </b>exampleToken <br />"
+        assertThat(result).isEqualTo(expected)
+    }
+
+    @Test
+    fun testFormatByteCount_zeroBytes() {
+        val resultNonSi = FormatUtils.formatByteCount(0, false)
+        val resultSi = FormatUtils.formatByteCount(0, true)
+        val expected = "0 B"
+        assertThat(resultNonSi).isEqualTo(expected)
+        assertThat(resultSi).isEqualTo(expected)
+    }
+
+    @Test
+    fun testFormatByteCount_oneKiloByte() {
+        testFormatByteCount(1024L, "1.0 kB", "1.0 KiB")
+    }
+
+    @Test
+    fun testFormatByteCount_oneKiloByteSi() {
+        testFormatByteCount(1023L, "1.0 kB", "1023 B")
+    }
+
+    private fun testFormatByteCount(
+        byteCountToTest: Long,
+        expectedSi: String,
+        expectedNonSi: String
+    ) {
+        val resultNonSi = FormatUtils.formatByteCount(byteCountToTest, false)
+        val resultSi = FormatUtils.formatByteCount(byteCountToTest, true)
+        assertThat(resultNonSi).isEqualTo(expectedNonSi)
+        assertThat(resultSi).isEqualTo(expectedSi)
+    }
+
+    @Test
+    fun formatXml_emptyString() {
+        assertThat(FormatUtils.formatXml("")).isEmpty()
+    }
+
+    @Test
+    fun formatXml_properXml() {
+        val xml =
+            """
+            <example>value</example>
+            """.trimIndent()
```

Comment:
Not sure that this variable should also be with `trimIndent()`

Reply:
Yes, it should otherwise there will be several white spaces before.
Having it on a single line won't work as well as there will be a missing `\n` at the end.